spec/spec.md

```diff
@@ -66,7 +66,9 @@ _Credential Manifests_ are a resource format that defines preconditional require
     }
   },
   "credential": {
-    "schema": "https://schema.org/EducationalOccupationalCredential",
+    "schema": {
```

Comment:
The PE `schema` property is an array of objects (that change was made many months ago in response to LD folks wanting multiple type URIs)

Reply:
updated